tests/plugins/imagebase/features/manual/captionbasicstyles.html

```diff
@@ -0,0 +1,51 @@
+<head>
+	<link rel="stylesheet" href="/apps/ckeditor/contents.css">
+</head>
+<body>
+	<h1>Classic editor</h1>
+
+	<div id="classic">
+		<p>Widget without caption:</p>
+		<figure>
+			<img src="../../../image2/_assets/foo.png" alt="foo">
+		</figure>
+
+		<p>Widget with caption:</p>
+		<figure>
+			<img src="../../../image2/_assets/foo.png" alt="foo">
+			<figcaption>Test caption</figcaption>
+		</figure>
+	</div>
+
+	<h1>Divarea</h1>
+
+	<div id="divarea">
+		<p>Widget without caption:</p>
+		<figure>
+			<img src="../../../image2/_assets/foo.png" alt="foo">
+		</figure>
+
+		<p>Widget with caption:</p>
+		<figure>
+			<img src="../../../image2/_assets/foo.png" alt="foo">
+			<figcaption>Test caption</figcaption>
+		</figure>
+	</div>
+
+	<script>
+		var cfg = {
```

Comment:
This test does not work (the divarea editor to be precise) on IE8. AFAIR, `imagebase` is not supposed to work on IE8 so it should be ignored probably.